src/05-derivatives/01-introduction.md

```diff
@@ -97,24 +122,13 @@ derivative includes REQUIRED metadata fields in which case a JSON file is also
-
-### CoordinateSystem key allowed values
-
-In addition to values defined in
-[Appendix VII Table "Template based Coordinate Systems"](../99-appendices/08-coordinate-systems.md).
-
-| **Value name** | **Description**                                                                                                             |
-| -------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| Unknown        | The coordinate system is unknown.                                                                                           |
-| Custom         | A custom coordinate system that is not in alignment (dimensions, axis orientation, unit) with any device coordinate system. |
+| **Key name**   | **Description**                                                                                                                                                                                                                                                                                                                                       |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Description    | RECOMMENDED. Free-form natural language description of the nature of the file.                                                                                                                                                                                                                                                                        |
+| Sources        | OPTIONAL. A list of paths relative to dataset root pointing to the file(s) that were directly used in the creation of this derivative. For example in a chain of A->B->C, “C” should only list “B” as Sources, and “B” should only list “A” as Sources. However in case X and Y jointly contribute to Z, then “Z” should list “X” and “Y” as Sources. |
+| RawSources     | OPTIONAL. A list of paths relative to dataset root pointing to the BIDS-Raw file(s) that were used in the creation of this derivative.                                                                                                                                                                                                                |  |
+| ReferenceMap   | REQUIRED when a custom template or file is used. A path to a file that was used as, or can be used as, a reference image for determining the coordinate space of this file. The path should start with a `/` and should be relative to the root of the dataset.                                                                                       |
```

Comment:
Is it possible for this to be a list, e.g., in the case of CIFTIs?

Reply:
i would think so, and we should clarify.

Reply:
I was thinking about it and shouldn't a `ReferenceMap` for a CIFTI file be another single CIFTI file?

Reply:
@edickie

Reply:
what I call `T1wfsLR` is the that I think you asked about in another thread, is the hardest. 

example: (for now)
```
sub-{subject}/anat/sub-{subject}_hemi-L_space-T1wfsLR32k_pial.surf.gii
{
ReferenceMap : ['./sub-{subject}_desc-preproc_T1w.nii.gz', 'fsLR32k']
}
```

is this making any sense.
P.S. I tried to list some cifti/hcp derivatives and what there `space` would be in this sheet https://docs.google.com/spreadsheets/d/1uJKQWoqGFD0FJE8Tm9hQVFj2-xOgo5zN1dO-ALwkT0M/edit?usp=sharing

Reply:
I'm not sure you intended to reply here, but let us roll with it :)

So subcortical part of the cifti file is in the native participant space, but cortical/surface part is transformed into a common template space? Is this common? What is it used for?

Reply:
Also, more to the point. I don't think it makes sense to have one cifti file reference.  There's 4 template files in template flow (L and R sphere + the MNI template + the subcortical parcellation template that end up being used..) I can't picture a nice what to combined those into 1 file...

Reply:
templateflow is young and still evolving so I am not sure it is the best reference. 

I know that you can definitely pack the two surfaces and subcortical part of the template in a CIFTI file. I am not an expert, but I thought that in connectome workbench one could easily load such CIFTI file and use it as a background to visualize another CIFTI file in the same space. Is that correct?

Would be great to get a clarification on the `T1wfsLR`. Thanks in advance.

Reply:
CIFTI files hold surface mapped data (scalar and timeseries) but the geometry (surf.gii) files have always needed to be loaded as two (or more) separate files in order to work with the data. also a volume template (i.e. T1w.nii.gz) was always loaded together as a background as well.

..actually, workbench gets around this by listing all surfaces and volumes that are in the same space within a ".spec" file (that workbench reads to load all files at once). We could recommend that the .json instead points to a ".spec" file for cifti?

Reply:
urg..yeah I see. These use hybrid spaces cases probably need their own table in the spec..

Reply:
Thanks for clarifying.

The most useful case for this (correct me if I am wrong) I can see is the `native` hybrid space (combination of `native` volume and `fsnative` surface). I think a list is better than adding a new data type (`.space`). This will only work if we require that when `space` is set to `native` for a CIFTI file `ReferenceMap` needs to be a 4 element list that includes surfaces for two hemispheres, volumetric template, and a subcortical parcellations (all valid BIDS derivatives files). Alternatively, we can prescribe a dictionary in such a situation. WDYT?